HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -189,4 +205,9 @@ def customizeHLTforCMSSW(process, menuType="GRun"):
     process = customiseFor18559(process)
     process = customiseFor19098(process)
 
+    if (menuType == "GRun2016"):
+        process = customiseFor19181_pixel_phase0(process)
+    else:
+        process = customiseFor19181_pixel_phase1(process)
```

Comment:
Run1 workflows use a stripped HLT menu with nearly no physics.
From the HLT technical point this looks ok, but I can't judge the physics/detector...